### DISCUSSION TITLE
Is it possible to use LEADER key followed by multiple keys?

### DISCUSSION THREAD

**Discussion by sommerper** [wezterm/wezterm#4997] [Q&A]
### What Operating System(s) are you running on?

Linux X11

### WezTerm version

20240210-173228-67bdc06d

### Ask your question!

I'd like to press a key sequence after the leader key so that for example
`LEADER + n + t` would open a new tab and `LEADER + n + p` would open a new pane.

I tried something like this:
```
table.insert(mykeys, {
  key = {'n', 'p'},
  -- key = 'n',
  mods = 'LEADER',
  action = wezterm.action.SplitPane {
    direction = 'Left',
    size = { Percent = 50 },
  },
})
```

But that doesn't work. But reading the docs it's also not mentioned so I'm not sure if it is even implemented.

**Comment by Geerayef** [comment]
Hello!

I was looking for exactly what you described, and fount this: https://wezterm.org/config/keys.html#raw-key-assignments.

I've had mixed results using the "raw:" mapping, but it might work for your needs.

**Reply by bew** [reply]
No that won't help for this wanted behavior. Raw keys are still for a single key

**Reply by Geerayef** [reply]
Thank you for the clarification!

**Comment by bew** [comment]
Yes you can, take a look at https://wezterm.org/config/key-tables.html
